tests/models/test_semantic_segmentation.py

```diff
@@ -428,7 +441,7 @@ def test_functional_segmentor_merging(tmp_path):
     _rm_dir(save_dir)
     os.mkdir(save_dir)
 
-    # * with out of bound location
+    # * without of bound location
```

Comment:
Is this comment not say "with an out of bounds location"?

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/618/commits/cd4acbb590f4021399117ccf9d90ff3963225818